src/App.js

```diff
@@ -7,6 +7,7 @@ import PrivateRoute from './utils/privateRoute';
 import Profile from './pages/profile/profile';
 import Main from './pages/main/main';
 import DemoJWT from './pages/jwt/jwt.jsx';
+import AdminTestPage from './pages/adminDistribution/tests-page-admin.jsx';
```

Comment:
why not 
./pages/AdminDistribution/AdminDistribution.jsx

or 

./pages/Tests-page-admin/Tests-page-admin.jsx?